new-log-viewer/src/components/StatusBar/index.tsx

```diff
@@ -38,15 +39,17 @@ const StatusBar = () => {
                 Status message
             </Typography>
             <Button
+                color={"primary"}
                 size={"sm"}
+                variant={"soft"}
                 onClick={handleCopyLinkButtonClick}
             >
-                Log Event
-                {" "}
+                {"Log Event: "}
```

Comment:
Do we need the colon? It makes it inconsistent with "Log level" where we can't add the colon.